packages/admin-next/dashboard/src/routes/products/common/upload-media-form-item.tsx

```diff
@@ -0,0 +1,87 @@
+import { useTranslation } from "react-i18next"
+import { FileType, FileUpload } from "../../../components/common/file-upload"
+import { Form } from "../../../components/common/form"
+
+const SUPPORTED_FORMATS = [
+  "image/jpeg",
+  "image/png",
+  "image/gif",
+  "image/webp",
+  "image/heic",
+  "image/svg+xml",
+]
+
+const SUPPORTED_FORMATS_FILE_EXTENSIONS = [
+  ".jpeg",
+  ".png",
+  ".gif",
+  ".webp",
+  ".heic",
+  ".svg",
+]
+
+export const UploadMediaFormItem = ({
+  form,
+  append,
+}: {
+  form: any
+  append: any
```

Comment:
missing types

Reply:
also needs to follow the filesystem convention for this one

Reply:
@kasperkristensen what do you mean by the filesystem convention?